tests/pages/firefox/ios.py

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+from pages.regions.modal import Modal
+
+
+class IOSPage(FirefoxBasePage):
+
+    _url = '{base_url}/{locale}/firefox/ios'
+
+    _app_store_button_locator = (By.CSS_SELECTOR, '#intro .appstore-badge')
+    _send_to_device_button_locator = (By.CSS_SELECTOR, '#intro .send-to-device .send-to')
+
+    @property
+    def is_app_store_button_displayed(self):
+        return self.is_element_displayed(self._app_store_button_locator)
+
+    @property
+    def is_send_to_device_button_displayed(self):
+        return self.is_element_displayed(self._send_to_device_button_locator)
+
+    def click_get_it_now(self):
+        modal = Modal(self)
+        self.find_element(self._send_to_device_button_locator).click()
```

Comment:
This is a bit confusing as there's a 'Get it now' button and then a 'Send' button. It might simply be that we need to rename this locator, or perhaps it will become more clear when you've brought in the changes from #3808.